src/components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -85,6 +94,26 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
 
   const [state, dispatch] = useReducer(resourceFormReducer, initialState);
 
+  const { data: facilitiesResponse } = useQuery<
```

Comment:
improve naming, response could be confused with the fetch's Response object. data refers to the response body and not the response as a whole.

```suggestion
  const { data: facilities } = useQuery<
```